lockfile/src/csharp.rs

```diff
@@ -248,14 +248,24 @@ mod tests {
     #[test]
     fn lock_parse_csproj() {
         let pkgs = CSProj.parse(include_str!("../../tests/fixtures/sample.csproj")).unwrap();
+        assert_eq!(pkgs.len(), 6);
 
-        assert_eq!(pkgs.len(), 5);
-        assert_eq!(pkgs[0].name, "Microsoft.NETFramework.ReferenceAssemblies");
-        assert_eq!(pkgs[0].version, PackageVersion::FirstParty("1.0.0".into()));
+        let expected_pkgs = [
+            Package {
+                name: "Microsoft.NETFramework.ReferenceAssemblies".into(),
+                version: PackageVersion::FirstParty("4.5.0".into()),
```

Comment:
I'm not sure why the version was changed here... It's `1.0.0` in `sample.csproj`

```suggestion
                version: PackageVersion::FirstParty("1.0.0".into()),
```

Reply:
Oh. This version was from the old check of `pkgs.last().unwrap()`. But the package name is from `pkgs[0].name`...

Anyways, I'll leave you to take care of that however you see fit.